jaseci_serv/jaseci_serv/jac_api/views.py

```diff
@@ -73,18 +77,40 @@ def log_request_stats(self, request):
             db_touches = self.caller._h.db_touch_count
             red_touches = self.caller._h.red_touch_count
             touch_kb = self.caller._h.mem_size()
-        logger.info(
-            str(
-                f"API call to {Cc.TG}{type(self).__name__}{Cc.EC}"
-                f" completed in {Cc.TY}{tot_time:.3f} seconds{Cc.EC}"
-                f" touched {Cc.TY}{touch_count}{Cc.EC} mem /"
-                f" {Cc.TY}{red_touches}{Cc.EC} redis /"
-                f" {Cc.TY}{db_touches}{Cc.EC} db "
-                f" ({Cc.TY}{touch_kb:.1f}kb{Cc.EC}) and"
-                f" saving {Cc.TY}{save_count}{Cc.EC} objects."
-            )
+
+        res_peek = str(api_result)[:256]
+        log_str = str(
+            f"API call to {Cc.TG}{type(self).__name__}{Cc.EC}"
+            f" completed in {Cc.TY}{tot_time:.3f} seconds{Cc.EC}"
+            f" touched {Cc.TY}{touch_count}{Cc.EC} mem /"
+            f" {Cc.TY}{red_touches}{Cc.EC} redis /"
+            f" {Cc.TY}{db_touches}{Cc.EC} db "
+            f" ({Cc.TY}{touch_kb:.1f}kb{Cc.EC}) and"
+            f" saving {Cc.TY}{save_count}{Cc.EC} objects."
+            f" From {self.caller.name}:{self.caller.jid}."
```

Comment:
Move this to the top of the message and make it green.

Reply:
Strip out the escape special characters from the message before sending it to elastic.

Reply:
Done.